### PR TITLE
Add Bread Crumbs to Journal Page

### PR DESCRIPTION
**Pull request by BessieSteinberg**

### CODE REVIEW

File: src/components/BreadCrumbs/BreadCrumbs.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  div {
+    display: inline-block;
+  }
+
+  li {
+    display: inline;
+
+    .bread-crumb-arrow {
+      font-size: 16px;
```
[the remark this answers is not in the document]
Comment (BessieSteinberg) [reply src/components/BreadCrumbs/BreadCrumbs.scss]: thanks - i'll take it out

File: src/App.scss [hunk 2]
```diff
@@ -73,10 +73,13 @@ h5{
...
-  font-weight: normal;
   line-height: 1.5;
   color: $dark-grey;
 }
 p.muted-text {
   color: $slate-grey;
 }
+a{
+  font-family: "Open Sans";
+  color: $deep-sea-blue;
```
[the remark this answers is not in the document]
Comment (BessieSteinberg) [reply src/App.scss]: haha yeah, i think we probably don't want users to change the link color? I think we can assume we don't want users to change the link color unless a partner specifies that

File: src/components/BreadCrumbs/index.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+          </Link>
+        </li>
+      ))
+  );
+}
+
+const BreadCrumbs = props => (
+  <ul>
+    <li>
+      <Link className="bread-crumb-link" to={`/${props.journalId}/about/`}>
```
[the remark this answers is not in the document]
Comment (BessieSteinberg) [reply src/components/BreadCrumbs/index.jsx]: I'm not really sure - I could make "home" be the name of the journal so we don't really have to make the choice of "what is home"

[the remark this answers is not in the document]
Comment (bill-filler) [reply src/components/BreadCrumbs/index.jsx]: I dont think we want the Journal About Page to show up in the breadcrumb list, even though that is technically the parent

File: src/components/BreadCrumbs/index.jsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        Home
+      </Link>
+    </li>
+    {PageList(props.ancestorPages, props.journalId)}
```
[the remark this answers is not in the document]
Comment (BessieSteinberg) [reply src/components/BreadCrumbs/index.jsx]: Does that mean i should convert BreadCrumbs into a proper React.Component?

**Pull request merged**